src/Turn.js

```diff
@@ -13,16 +15,13 @@ class Turn {
   }
 
   evaluateGuess() {
-    return this.guess === this.card['correctAnswer'];
+    return this.guess === this.card.correctAnswer;
   }
 
   giveFeedback() {
-    if (this.guess === this.card['correctAnswer']) {
-      return 'Correct!';
-    } else {
-      return 'Incorrect!';
-    }
+    return (this.guess === this.card['correctAnswer']) ? 'Correct!' : 'Incorrect!';
```

Comment:
Not sure why I changed one of these to dot notation and not the other... I should change both to be the same.